src/Surfnet/Stepup/Configuration/InstitutionConfiguration.php

```diff
@@ -172,7 +172,9 @@ public function configureShowRaaContactInformationOption(ShowRaaContactInformati
 
     public function updateAllowedSecondFactorList(AllowedSecondFactorList $allowedSecondFactorList)
     {
-        if ($this->allowedSecondFactorList->equals($allowedSecondFactorList)) {
+        if ($this->allowedSecondFactorList !== null
```

Comment:
It should be documented that this can only be `null` due to pre-existing InstitutionConfigurations where this functionality did not yet exist.